backend/app.py

```diff
@@ -20,11 +27,15 @@ def main():
 
 @app.before_request
 def before_request():
-    request.db = psycopg2.connect(host=os.environ.get('DB_HOST'), port=os.environ.get('DB_PORT'), database=os.environ.get('DB_NAME'), user=os.environ.get('DB_USER'), password=password_decoded) if os.environ.get('IS_LOCAL') else pymysql.connect(host=os.environ.get('DB_HOST'), database=os.environ.get('DB_NAME'), user=os.environ.get('DB_USER'), password=password_decoded)
+    if os.environ.get('IS_LOCAL'):
+        request.db = psycopg2.connect(host=os.environ.get('DB_HOST'), port=os.environ.get('DB_PORT'), database=os.environ.get('DB_NAME'), user=os.environ.get('DB_USER'), password=password_decoded)
```

Comment:
same thing here instead of reading values from `os.environ` read them from `app.config`